rmgpy/chemkin.pyx

```diff
@@ -1035,14 +1035,16 @@ def load_chemkin_file(path, dictionary_path=None, transport_path=None, read_comm
     # Read in the thermo data from the thermo file        
     if thermo_path:
         with open(thermo_path, 'r') as f:
+            previous_line = f.tell()
             line0 = f.readline()
             while line0 != '':
                 line = remove_comment_from_line(line0)[0]
                 line = line.strip()
```

Comment:
```suggestion
                line = remove_comment_from_line(line0)[0].strip()
```

Reply:
Added this in (but used commit --amend and a force push to keep it all in the same commit)